SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -1211,6 +1231,11 @@ FReply FSpatialGDKEditorToolbarModule::OnStartCloudDeployment()
 		return FReply::Unhandled();
 	}
 
+	if (SpatialGDKSettings->IsAutoGenerateConfigEnabled())
```

Comment:
You might want to explicitely check for the return value since this method is not returning a boolean.

Reply:
This one is returning a bool though. It's a bit confusing because there's a function with the same name on `SSpatialGDKCloudDeploymentConfiguration` that's returning a checkbox state.

Reply:
Will certainly do that but can you please provide some more detail on why this is the case? It appears to me that the function is returning a boolean?

Reply:
Yes I have noticed that and changed the name of the function from the Setttings to ShouldAutoGenerateCloudLaunchConfig

Reply:
Oh, right, that was confusing.